packages/client/hmi-client/src/components/workflow/util.ts

```diff
@@ -134,3 +139,20 @@ export const getGraphDataFromDatasetCSV = (
 
 	return graphData;
 };
+
+export function getSelectedOutput<S>(node: WorkflowNode<S>) {
```

Comment:
This looks to be a repeat of `getActiveOutput` in `/services/operator.ts`
Worth double checking and keeping one or the other

Reply:
Nice catch, we had total 3 functions doing exactly same thing lol. I removed other two.